app/routes/calendar.py

```diff
@@ -0,0 +1,60 @@
+from app.models.models import db, Calendar
+from flask_restx import Resource, Namespace, fields
+import datetime
+
+api = Namespace('calendar', description='Calendar operations')
+
+# expected information
+model = api.model("Calendar",
+                {
+                    "start_date": fields.Date(required=True, description="Calendar start date"),
+                    "end_date": fields.Date(required=True, description="Calendar end date"),
+                    "location_id": fields.Integer(required=True, description="Calendar location"),
```

Comment:
can you please add` example = 1` for  location_id and user_id? It can be done on a separate change

Reply:
Ok, I'm on it